default/scripting/common/priorities.macros

```diff
@@ -111,6 +111,9 @@ LATE_AFTER_ALL_TARGET_MAX_METERS_PRIORITY
 FOCUS_CHANGE_PENALTY_PRIORITY
 '''600'''
 
+COLONY_INDEPENDENCE_PRIORITY
+'''650'''
+
```

Comment:
Is it necessary to introduce another priority level? Can be done with existing levels? Maybe using `METER_OVERRIDE_PRIORITY` is enough.

Reply:
probably its not necessary.